mpas_analysis/config.default

```diff
@@ -266,6 +266,25 @@ sshSubdirectory = SSH
 argoSubdirectory = ARGO
 schmidtkoSubdirectory = Schmidtko
 woceSubdirectory = WOCE
+no3Subdirectory = BGC/NO3
+po4Subdirectory = BGC/PO4
+sio3Subdirectory = BGC/SiO3
+o2Subdirectory = BGC/O2
+co2_gas_fluxSubdirectory = BGC/CO2_gas_flux
+dicSubdirectory = BGC/DIC
+alkSubdirectory = BGC/ALK
+ph_3dSubdirectory = BGC/pH_3D
+pco2surfaceSubdirectory = BGC/pCO2surface
+chlSubdirectory = BGC/Chl
+
+# first and last year of SST observational climatology (preferably one of the
+# two ranges given below)
+# values for preindustrial
+sstClimatologyStartYear = 1870
+sstClimatologyEndYear = 1900
+# alternative values for present day
+#sstClimatologyStartYear = 1990
+#sstClimatologyEndYear = 2011
```

Comment:
@bradyrx, I think this is a glitch from a rebase.  This stuff got moved to a different section of the config file and should be removed here.